src/NS1_request.js

```diff
@@ -114,7 +124,13 @@ function apply_data(query, files) {
 function create_promise() {
   return new Promise((resolve, reject) => {
     this.request.end((err, response) => {
-      if (err) reject(handle_error.call(this, err, response))
+      if (err) {
+        reject(handle_error.call(this, err, response))
```

Comment:
the only difference with this line is that I removed the inline if statement to add the else clause.